SQL/0000-00-03-ConfigTables.sql

```diff
@@ -250,3 +251,7 @@ INSERT INTO Config (ConfigID, Value) SELECT ID, "/issue_tracker" FROM ConfigSett
 INSERT INTO Config (ConfigID, Value) SELECT ID, 0 FROM ConfigSettings cs WHERE cs.Name="ComputeDeepQC";
 INSERT INTO Config (ConfigID, Value) SELECT ID, 'prod' FROM ConfigSettings cs WHERE cs.Name="mriConfigFile";
 INSERT INTO Config (ConfigID, Value) SELECT ID, 'environment' FROM ConfigSettings cs WHERE cs.Name="EnvironmentFile";
+INSERT INTO Config (ConfigID, Value) SELECT cs.ID, GROUP_CONCAT(mst.Scan_Type) FROM ConfigSettings cs JOIN mri_scan_type mst WHERE cs.Name="compute_snr_modalities" AND mst.ID=41;
+INSERT INTO Config (ConfigID, Value) SELECT cs.ID, GROUP_CONCAT(mst.Scan_Type) FROM ConfigSettings cs JOIN mri_scan_type mst WHERE cs.Name="compute_snr_modalities" AND mst.ID=44;
+INSERT INTO Config (ConfigID, Value) SELECT cs.ID, GROUP_CONCAT(mst.Scan_Type) FROM ConfigSettings cs JOIN mri_scan_type mst WHERE cs.Name="compute_snr_modalities" AND mst.ID=45;
+INSERT INTO Config (ConfigID, Value) SELECT cs.ID, GROUP_CONCAT(mst.Scan_Type) FROM ConfigSettings cs JOIN mri_scan_type mst WHERE cs.Name="compute_snr_modalities" AND mst.ID=46;
```

Comment:
`GROUP_CONCAT` not needed as there is only one `Scan_type` value for each `ID`. 
Calling the `Scan_type` name from the DB using artificial `ID` number is a bad solution. You can't predict ID (auto_increment) number for an installation. Auto_increment might be different than 1 (good practice when clustering)
```suggestion
INSERT INTO Config (ConfigID, Value) SELECT ID, `pd` FROM ConfigSettings WHERE Name="compute_snr_modalities";
```
 same for lines 254-256

Reply:
done in the next commit